### DISCUSSION TITLE
Slots docs

### DISCUSSION THREAD

**Discussion by louiss0** [withastro/docs#1771] [Ideas]
I need to inform you that you did not properly show how Slots render works. The image below shows only arguments to the function.

![image](https://user-images.githubusercontent.com/29028601/194961815-75ee400c-8a10-4fe3-8b5f-2a698a6d7450.png)

I now know that when you pass arguments into the array you get a new argument passed to the function for each value that was inside of the array. 

```
Astro.slots.render("default", ['10x', 'developer'])
```

```astro

<div>
{ (slotProp, slotProp2)=> (
<div>
{slotProp} <span>{slotProp2}</span>
</div>
) }
</div>

```

This page does not tell anyone this at all

**Comment by ArmandPhilippot** [comment]
We're closing stale and outdated discussion to keep only the discussions still relevant to the current project version and/or the current docs state.

If you think this is a mistake and that this discussion should remain open, please feel free to ping me, I'll take a second look at it.

Thank your for your contribution!
